README.md

```diff
@@ -41,8 +41,19 @@ In order for an insurance company to make money, it needs to collect more in yea
 
 ### Personal Rationale
 
-(add my biography)(add that I would focus on descriptive statistics in depth as a beginner)
-(add that I would include lots of data visualization as that is my favorite part of data analytics)
+My name is Grace Uche. I am a Professional Registered Health Information Administrator with a Master Degree in Business Administration. I’m also a Licensed Health and Life Insurance Broker. As a Health Information Management professional, for over 15 years, I have had the opportunity to practice and work in different healthcare settings. Most of my work has been in educating and training  students in both Associates and Bachelors degrees in the Health Information Management programs to successfully complete the program, sit and pass any of the national certificate exams, and be able to obtain a meaningful job in the field.
+
+The  quest for knowledge drove me into Health and Life Insurance because I wanted to understand how insurance company carriers operate, factors that determine insured insurability and coverage suitability, as well as to understand those situations that might deter individuals from not being able to get coverage.
```

Comment:
```suggestion
The  quest for knowledge drove me into Health and Life Insurance because I wanted to understand how insurance companies operate, factors that determine insurability and coverage suitability. Furthermore, I wanted to explore situations that might deter individuals from getting coverage.
```